draft-ietf-quic-transport.md

```diff
@@ -2439,7 +2439,9 @@ indicates that the peer has migrated to that address.
 If the recipient permits the migration, it MUST send subsequent packets
 to the new peer address and MUST initiate path validation ({{migrate-validate}})
 to verify the peer's ownership of the address if validation is not already
-underway.
+underway.  If the receipient has no available connection IDs, it will not be
+able to send on the new path until the peer provides one;
+see {{migration-linkability}}.
```

Comment:
```suggestion
underway. If the recipient has no connection IDs available for the peer, it will
not be able to send anything on the new path until the peer provides one; see
{{migration-linkability}}.
```